nuxeo-drive-client/nxdrive/engine/engine.py

```diff
@@ -367,6 +367,8 @@ def suspend(self):
     def unbind(self):
         self.stop()
         try:
+            # Unregister the favorite link for Nuxeo Drive upon disconnecting.
+            self._manager.get_osi().unregister_folder_link(self._manager._get_default_nuxeo_drive_name())
```

Comment:
I have updated code to use engineId instead of productId to register shortcut. Now for multiple account, two shortcuts will be available.